src/components/Dropdown/V2/Dropdown.MenuContainer.tsx

```diff
@@ -418,41 +427,40 @@ export class MenuContainer extends React.PureComponent<Props> {
 
     return (
       <div className="DropdownV2MenuContainerRoot" ref={this.setWrapperNode}>
-        {isOpen && (
-          <Portal
-            id={this.id}
-            onOpen={this.onPortalOpen}
-            onClose={this.onPortalClose}
+        <MenuPortal
```

Comment:
Always renders vs. conditional rendering on `isOpen`. Instead, the `children` content will conditionally render based on `isOpen={isOpen}`